pycbc/events/stat.py

```diff
@@ -113,7 +113,44 @@ def single(self, trigs):
         """
         return get_newsnr(trigs)
 
-    def coinc(self, s0, s1, slide, step):
+    def coinc(self, s0, s1, slide, step): # pylint:disable=unused-argument
+        """Calculate the coincident detection statistic.
+
+        Parameters
+        ----------
+        s0: numpy.ndarray
+            Single detector ranking statistic for the first detector.
+        s1: numpy.ndarray
+            Single detector ranking statistic for the second detector.
+        slide: (unused in this statistic)
+        step: (unused in this statistic)
+
+        Returns
+        -------
+        numpy.ndarray
+            Array of coincident ranking statistic values
+        """
+        return (s0**2. + s1**2.) ** 0.5
+
+class NewSNRSGStatistic(Stat):
```

Comment:
Think it would be much more concise and simpler to make this a subclass of NewSNRStatistic and just redefine the single() method, as already done with PhaseTDExpFitSGStatistic (see around line 500).

Reply:
Makes sense. I just pushed a patch to do this.